addons/l10n_ec/data/account_tax_report_data.xml

```diff
@@ -87,6 +87,42 @@
                             </record>
                         </field>
                     </record>
+                    <record id="tax_report_line_parent_line_report_7" model="account.report.line">
+                        <field name="name">Ventas locales (excluye activos fijos) gravadas tarifa 5%</field>
+                        <field name="children_ids">
+                            <record id="tax_report_line_104_425" model="account.report.line">
+                                <field name="name">Valor bruto(425)</field>
+                                <field name="code">c425_104</field>
+                            <field name="expression_ids">
```

Comment:
indentation?

Reply:
hmm... this identation is not shown  in two code editor

Reply:
It's fixed @kenluzuriaga0. It's ready to merge @jco-odoo